types/uikit/README.md

```diff
@@ -1,61 +0,0 @@
-[![uikit banner](https://cloud.githubusercontent.com/assets/321047/21769911/474d7d9e-d681-11e6-9fe0-d95f8ccfd3a9.jpg)](http://getuikit.com/)
```

Comment:
Are these deletions related to the rest of the change?

Reply:
Yeah, they fail the new check https://github.com/DefinitelyTyped/DefinitelyTyped/runs/1420626076#step:6:73

because they https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/uikit/README.md

link to non-existent files, e.g. https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/uikit/CONTRIBUTING.md

I think what's going on is that the README is copied from the underlying package https://github.com/uikit/uikit/blob/develop/README.md

without the ancillary files. I opted to remove rather than fix it because it documents the underlying package vs. the types.

Reply:
Makes sense.